docs/listings/baby_fuzzer/listing-04/src/main.rs

```diff
@@ -77,7 +77,7 @@ fn main() {
 
     /* ANCHOR: generator */
     // Generator of printable bytearrays of max size 32
-    let mut generator = RandPrintablesGenerator::new(32).unwrap();
+    let mut generator = RandPrintablesGenerator::new(NonZeroUsize::new(32).unwrap());
```

Comment:
I don't like this change, the mutator is not a performance-critical path, let's just keep it be a normal usize and check the value on creation

Reply:
In any case, use the generic `NonZero::new()` instead - it's shorter.

Reply:
I don't understand. 
I thought the point of your previous PR is that you want compile-time check instead of debug_ssert so that users won't pass 0 in inappropriate place and get runtime errors.
but now you say the opposite.

Reply:
Here, `new` is only called once. If the new "unwraps" internally or an external `NonZero` call does it makes no difference for the user nor for performance - but now the API is more ugly..